src/amber/cli/templates/app/README.md.ecr

```diff
@@ -1,52 +1,84 @@
 # <%= @name %>
 
-Your description here
+A new project using [Amber Framework](https://amberframework.org/)
 
 ## Installation
 
+Install back-end and front-end dependencies.
+  
+```
+shards install
+npm install
+```
+
 Configure the `config/environments/development.yml` and set the `database_url` with your credentials to your database.
```

Comment:
Is this line still valid?, Should we add `amber encrypt` reference to this README too? What steps are needed to setup `production.yml`? What about `.encryption_key`?

Reply:
That might live nicely in a Deployment section

Reply:
@robacarp I added a link to `amber encrypt` documentation 👍